### PR TITLE
partner_prospect

### PR DESCRIPTION
**Pull request by MAbdelbaset**

### CODE REVIEW

File: partner_prospect/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Esther Martin - AvanzOSC
+# (c) 2015 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Partner Prospect",
+    "version": "8.0.1.0.0",
```
Comment (RoelAdriaans) [inline partner_prospect/__openerp__.py:8]: Update to 10.0

**Review by RoelAdriaans — CHANGES_REQUESTED** [review]
Thanks for your work, but a couple of remarks: 

- You are updating  sale_quotation_number and partner_prospect in the same pull request, please make a seperate request for every module
- Squash commits
- Update version On modules
- Remove unwanted *pyc files

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Please respect module history by following the migration wiki:
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0

**Comment by rafaelbn** [comment]
Please @MAbdelbaset rebase and attend comments https://github.com/OCA/sale-workflow/pull/638#pullrequestreview-122564382 Thanks!

**Comment by rousseldenis** [comment]
Closing this as very old and not attended. Please reopen if needed.

**Pull request closed without merging**